Dockerfile

```diff
@@ -1,39 +1,47 @@
 FROM golang AS build
-ADD . /go/src/github.com/assetto-corsa-web/accweb
+
+LABEL maintainer="Kugel" \
+	  contributors="GillesDubois" \
+      version="latest" \
+      description="Kugel's accweb app inside docker !"
```

Comment:
Probably change this to "Assetto Corsa Competizione Server Management Tool via Web Interface.". I don't need to have my name in it :)